privacy-snapshot.json

```diff
@@ -2,6 +2,7 @@
   "accounts.api.cx.metamask.io",
   "acl.execution.metamask.io",
   "api.devnet.solana.com",
+  "api.etherscan.io",
```

Comment:
This API is used by the petnames system but had not been added to the privacy snapshot yet since it doesn't resolve on testnet.